src/test/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/controller/SearchCsipRecordsIntTest.kt

```diff
@@ -92,6 +94,15 @@ class SearchCsipRecordsIntTest : IntegrationTestBase() {
     val res = searchCsipRecords(searchRequest(prisonCode = SEARCH_PRISON_CODE, query = csip1.prisonNumber))
     assertThat(res.content.size).isEqualTo(1)
     res.content.first().verifyAgainst(csip1)
+
+    val res2 = searchCsipRecords(
```

Comment:
Could we pick a more descriptive name than `res2` for this? How does `res2` differ from `res` ?

Reply:
Yeah its the null character difference I wanted to highlight. The test name doesn't make this clear either. Outside of the context of this PR's title it might be a point of confusion why we have that second search at all. `res` is good as shorthand, but perhaps `resNullCharacter` or something to that effect would help identify its purpose. 

Alternatively maybe a separate test (or renaming this test) to explicitly point out "can handle null character searches" might help